tests/test_default_qubit.py

```diff
@@ -417,21 +417,21 @@ def test_apply_operation_three_wires_no_parameters(self, qubit_device_3_wires, t
         assert np.allclose(qubit_device_3_wires._state, np.array(expected_output), atol=tol, rtol=0)
 
 
-    @pytest.mark.parametrize("name,input,expected_output,par", [
-        ("BasisState", [1, 0, 0, 0], [0, 0, 1, 0], [[1, 0]]),
-        ("BasisState", [1/math.sqrt(2), 0, 1/math.sqrt(2), 0], [0, 0, 1, 0], [[1, 0]]),
-        ("BasisState", [1/math.sqrt(2), 0, 1/math.sqrt(2), 0], [0, 0, 0, 1], [[1, 1]]),
-        ("QubitStateVector", [1, 0, 0, 0], [0, 0, 1, 0], [[0, 0, 1, 0]]),
-        ("QubitStateVector", [1/math.sqrt(2), 0, 1/math.sqrt(2), 0], [0, 0, 1, 0], [[0, 0, 1, 0]]),
-        ("QubitStateVector", [1/math.sqrt(2), 0, 1/math.sqrt(2), 0], [0, 0, 0, 1], [[0, 0, 0, 1]]),
-        ("QubitStateVector", [1, 0, 0, 0], [1/math.sqrt(3), 0, 1/math.sqrt(3), 1/math.sqrt(3)], [[1/math.sqrt(3), 0, 1/math.sqrt(3), 1/math.sqrt(3)]]),
-        ("QubitStateVector", [1, 0, 0, 0], [1/math.sqrt(3), 0, -1/math.sqrt(3), 1/math.sqrt(3)], [[1/math.sqrt(3), 0, -1/math.sqrt(3), 1/math.sqrt(3)]]),
+    @pytest.mark.parametrize("name,expected_output,par", [
+        ("BasisState", [0, 0, 1, 0], [[1, 0]]),
+        ("BasisState", [0, 0, 1, 0], [[1, 0]]),
+        ("BasisState", [0, 0, 0, 1], [[1, 1]]),
+        ("QubitStateVector", [0, 0, 1, 0], [[0, 0, 1, 0]]),
+        ("QubitStateVector", [0, 0, 1, 0], [[0, 0, 1, 0]]),
+        ("QubitStateVector", [0, 0, 0, 1], [[0, 0, 0, 1]]),
+        ("QubitStateVector", [1/math.sqrt(3), 0, 1/math.sqrt(3), 1/math.sqrt(3)], [[1/math.sqrt(3), 0, 1/math.sqrt(3), 1/math.sqrt(3)]]),
+        ("QubitStateVector", [1/math.sqrt(3), 0, -1/math.sqrt(3), 1/math.sqrt(3)], [[1/math.sqrt(3), 0, -1/math.sqrt(3), 1/math.sqrt(3)]]),
```

Comment:
This test was changed because the default qubit state prep. operations can only be applied as the first operation now.